### DISCUSSION TITLE
In revelajs, how to align h2 when the text has a padding and a backgorund color (like a text-box)

### DISCUSSION THREAD

**Discussion by ManuelSpinola** [quarto-dev/quarto-cli#5692] [Q&A] [answered]
When I use the h2 with background color (like a text box) and use de css display: inline, the "text-box" autoamtically justify to the left.  How can I keep everything centered, the text and the "text-box".

.reveal section h2 {
  color: #fff;
  padding: 5px;
  border-radius: 10px;
  background-color: DodgerBlue;
  text-align: center;
  font-size: 1em;
  display: inline;
}

**Comment by mcanouil** [comment]
Please could you format your post?

Also, could you share a small "working" (reproducible) example to work with, _i.e._, a complete Quarto document or a Git repository? Thanks.

You can share a Quarto document using the following syntax, _i.e._, using more backticks than you have in your document (usually four ` ```` `).

`````md
````qmd
---
title: "Reproducible Quarto Document"
format: html
---

This is a reproducible Quarto document using `format: html`.
It is written in Markdown and contains embedded R code.
When you run the code, it will produce a plot.

```{r}
plot(cars)
```

The end.
````
`````

**Reply by ManuelSpinola** [reply]
````qmd
---
title: "Reproducible Quarto Document"
format:
  revealjs: 
    theme: [default, my_theme]
    self-contained: true
    slide-number: true

---

## Introduction

````

My question is regarding to the html formatting of the h2

This is my scss (my_theme)

.reveal section h2 {
color: #fff;
padding: 5px;
border-radius: 10px;
background-color: DodgerBlue;
text-align: center;
font-size: 1em;
display: inline;
}

**Comment by mcanouil** [comment, marked as answer]
Because of the various `display: block`, either you learn to tweedle with CSS until your figure out the proper cascade (not 100 % sure it's actually possible here, since I believe you want the background to only appear under the text).

<table>
<tr><th>Quarto document</th><th>HTML</th></tr>
<tr>
<td>

````qmd
---
title: "Reproducible Quarto Document"
format: revealjs
include-in-header:
  - text: |
      <style>
      .text-box {
        color: #fff;
        padding: 5px;
        border-radius: 10px;
        background-color: DodgerBlue;
        font-size: 1em;
      }
      .reveal .slides section h2 {
        text-align: center;
      }
      </style>
---

## [Introduction]{.text-box}

{{< lipsum 1 >}}

````

</td>
<td>
<img width="727" alt="image" src="https://github.com/quarto-dev/quarto-cli/assets/8896044/5af1d2c2-3db5-4e14-b143-63d3f3c6463d">
</td>
</tr>
</table>

**Reply by ManuelSpinola** [reply]
Thank you very much.  I will try also to figure it out with CSS.
